provisions/roles/api_poll/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+- name: Ensure EPEL is installed
+  yum: name=epel-release state=present
+  tags: jenkins
+
+- name: Install Wget, python-pip
+  yum: name={{ item }} state=present
+  with_items:
+    - wget
+    - python-pip
+  become: true
+  tags: jenkins
+
+- name: Configure jenkins requirements
+  shell: |
+      wget -O /etc/yum.repos.d/jenkins.repo http://pkg.jenkins-ci.org/redhat/jenkins.repo
+      rpm --import https://jenkins-ci.org/redhat/jenkins-ci.org.key
+  become: true
+  tags: jenkins
+
+- name: Install jenkins, java, jenkins-jjb
+  yum: name={{ item }} state=present
+  with_items:
+    - java
+    - jenkins
+  become: true
+  tags: jenkins
+
+- name: Restart Jenkins
+  systemd: name=jenkins state=restarted enabled=yes
+  become: true
+  tags: jenkins
+
+- name: Enable jenkins
```

Comment:
this is already done in just above step `enabled=yes`

please remove this task